check_stability.py

```diff
@@ -582,6 +622,11 @@ def get_parser():
                         # This is a workaround to get what should be the same value
                         default=os.environ.get("TRAVIS_REPO_SLUG").split('/')[0],
                         help="Travis user name")
+    parser.add_argument("--output-cap",
```

Comment:
Addressed this in 1501eac729102ad130a40d0f9d3120bdd49312ff. GitHub seems a little slow today, so you may have missed that